### ISSUE TITLE
Logcat says "GStreamer: Could not retrieve class org.freedesktop.gstreamer.GStreamer"

### ISSUE THREAD

**Issue by jnsw** [mrayy/mrayGStreamerUnity#56]
`10-14 04:46:23.563  7019  7043 I Unity   : initializing gstreamer
10-14 04:46:23.563  7019  7043 I Unity   : GStreamerCore:Ref()
10-14 04:46:23.563  7019  7043 I Unity   : CustomSBSPipelinePlayer:Start()
10-14 04:46:23.563  7019  7043 I Unity   : 
10-14 04:46:23.591  7019  7043 E GStreamer: Could not retrieve class org.freedesktop.gstreamer.GStreamer`

I used the Android Library ARM64 from [MRTunnelingPico](https://github.com/keli95566/MRTunnelingPico) (like mentioned [here](https://github.com/mrayy/mrayGStreamerUnity/issues/32#issuecomment-1220706909) and want deploy the project for evaluation on a S20+ with Unity 2021.3.31f1. I am using the Scenes from https://github.com/mrayy/mrayGStreamerUnity/tree/master/Unity/UnityTests/Assets/Scenes 

But I am getting the log above, what am I missing?

**Comment by jnsw** [comment]
I tried it now on my Quest to but it states:
10-17 01:01:05.682 20270 20298 E Unity   : EntryPointNotFoundException: Unable to find an entry point named 'mray_gst_customPlayerBlitImageNativeGLCall' in 'GStreamerUnityPlugin'. Tried the following entry points: Unable to get function 'mray_gst_customPlayerBlitImageNativeGLCall' because of 'The requested function was not found (0x06000001) dlerror() = undefined symbol: mray_gst_customPlayerBlitImageNativeGLCall

**Comment by edevdoge** [comment]
Hi! Did you solve this issue?

**Comment by jnsw** [comment]
No, I did not. Help is appreciated.

**Comment by edevdoge** [comment]
Did you also want to use gstreamer to receive udp stream on Oculus Quest? I've been trying to come up with a solution for a couple of days now and it's really hard

**Comment by jnsw** [comment]
yes, but my problem was i couldn even get the drivers loaded, hours wasted for nothing

**Comment by edevdoge** [comment]
Maybe you managed to find another solution to receive a UDP stream in Unity? Would be grateful to hear if smth is works, because I've already tried a lot of solutions.

**Comment by jnsw** [comment]
i think some here managed it, but not me
